src/main/java/crud/dao/impl/CarDaoJdBcImpl.java

```diff
@@ -0,0 +1,169 @@
+
+import static crud.dao.impl.DriverDaoJdBcImpl.setDriver;
+import static crud.dao.impl.ManufacturerDaoJdBcImpl.setManufacturer;
+
+import crud.dao.abstraction.CarDao;
+import crud.lib.DaoImpl;
+import crud.lib.DataBaseException;
+import crud.model.Car;
+import crud.model.Driver;
+import crud.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@DaoImpl
+public class CarDaoJdBcImpl implements CarDao {
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String query = "SELECT c.car_id FROM cars_drivers cd "
+                + "JOIN cars c on c.car_id = cd.cars_id "
+                + "WHERE cd.driver_id = ? AND c.deleted = FALSE";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, driverId);
+            ResultSet set = statement.executeQuery();
+            while (set.next()) {
+                Long carId = set.getObject(1, Long.class);
+                cars.add(get(carId).get());
+            }
+            return cars;
+        } catch (SQLException throwables) {
+            throw new DataBaseException("Could not get a list of cars by driver id "
+                    + driverId, throwables);
+        }
+    }
+
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (manufacturer_id, car_model) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = setUpdate(connection.prepareStatement(query,
+                        Statement.RETURN_GENERATED_KEYS), car)) {
+            statement.executeUpdate();
+            ResultSet set = statement.getGeneratedKeys();
+            if (set.next()) {
+                car.setId(set.getObject(1, Long.class));
+            }
+            return car;
+        } catch (SQLException throwables) {
+            throw new DataBaseException("Couldn't insert " + car + " into carsDB.", throwables);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String query = "SELECT c.car_id, m.manufacturer_id, c.car_model, "
+                    + "m.manufacturer_name, m.manufacturer_country FROM cars c "
+                + "JOIN manufacturers m ON m.manufacturer_id = c.manufacturer_id "
+                + "WHERE c.car_id = ? AND c.deleted = FALSE";
+        String getDriversQuery = "SELECT d.driver_id, d.driver_name, d.driver_license_number "
+                    + "FROM cars c "
+                + "JOIN cars_drivers cd on c.car_id = cd.cars_id "
+                + "JOIN drivers d on d.driver_id = cd.driver_id "
+                + "WHERE c.car_id = ? AND c.deleted = FALSE";
```

Comment:
You can select FROM cars_drivers and then you can remove JOIN with cars, in WHERE statement you can remove chech for deleted, because it checked in prev. query